mrp_project/models/project_project.py

```diff
@@ -11,11 +11,34 @@ class ProjectProject(models.Model):
 
     def _compute_manufacture_order_count(self):
         for record in self:
-            record.manufacture_order_count = len(record.manufacture_ids)
+
+            def count_all_children(mrp):
+                children = mrp._get_children()
+                count = len(children)
+                for child in children:
+                    count += count_all_children(child)
+                return count
+            
+            mrp_production_childs = sum(count_all_children(mrp) for mrp in record.manufacture_ids)
+            record.manufacture_order_count = len(record.manufacture_ids) + mrp_production_childs
```

Comment:
Not sure why is this needed. Once MOs have parent's project, they are all linked to project, so manufacture_ids will contain every MO linked to project, no matter if they were directly assigned or via an ancestor